src/nl/rubensten/texifyidea/inspections/latex/LatexFileNotFoundInspection.kt

```diff
@@ -35,20 +36,30 @@ open class LatexFileNotFoundInspection : TexifyInspectionBase() {
         for (command in commands) {
             // Only consider default commands with a file argument.
             val default = LatexCommand.lookup(command.name) ?: continue
-            val arguments = default.arguments
-            val parameters = command.parameterList
+            // Remove optional arguments from list of commands
+            val arguments = default.arguments.filter { it is RequiredArgument }
+                    .map { it as RequiredArgument }
+            // Remove optional parameters from list of parameters
+            val parameters = command.parameterList.filter { it.requiredParam != null }
```

Comment:
```suggestion
            val parameters = command.parameterList.filterNotNull()
```

Reply:
This change breaks the function. In the parameters list I need only required parameters. Only when in the arguments list and in the parameters list only required arguments are available the index of them matches every time.

Reply:
oh wow. my bad!